docs/management_api.md

```diff
@@ -34,6 +34,20 @@ This API follows the [ManagementAPIsService.RegisterModel](../frontend/server/sr
 ```bash
 curl -X POST  "http://localhost:8081/models?url=https://torchserve.pytorch.org/mar_files/squeezenet1_1.mar"
 
+{
+  "status": "Model \"squeezenet_v1.1\" Version: 1.0 registered with 0 initial workers. Use scale workers API to add workers for the model."
+}
+```
+Your model might be encrypted via [S3 SSE-KMS](https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingKMSEncryption.html). In this case, you need specify the following [environment variables](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/setup-credentials.html). 
```

Comment:
I would frame this a bit differently as

"If you'd like to serve an encrypted model then you need to setup S3 SSE-KMS with the following environment variables"